apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Generated by `createschema marketplace.Invoice 'number:Integer; property:Relationship:Property:PROTECT; unitType:Text; unitName:Text; accountNumber:Text; toPay:Decimal; items:Json; contact?:Relationship:Contact:SET_NULL; client?:Relationship:User:SET_NULL; clientName?:Text; clientPhone?:Text; clientEmail?:Text'`
+ */
+
+const Ajv = require('ajv')
+const addFormats = require('ajv-formats')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { AutoIncrementInteger } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { getGQLErrorValidator } = require('@condo/domains/common/schema/json.utils')
+const access = require('@condo/domains/marketplace/access/Invoice')
+const {
+    ERROR_INVALID_INVOICE_ROWS,
+    ERROR_NO_INVOICE_RECEIVERS,
+    ERROR_INVOICE_ALREADY_PAID,
+    INVOICE_STATUSES,
+    INVOICE_STATUS_DRAFT, INVOICE_STATUS_PAID,
+} = require('@condo/domains/marketplace/constants')
+
+const ERRORS = {
+    NO_RECEIVERS: {
+        code: BAD_USER_INPUT,
+        type: ERROR_NO_INVOICE_RECEIVERS,
+        message: 'Nobody sees this invoice. You must set property+unitType+unitName or accountNumber',
+    },
+    ALREADY_PAID: {
+        code: BAD_USER_INPUT,
+        type: ERROR_INVOICE_ALREADY_PAID,
+        message: 'Changing of paid invoice is forbidden',
+    },
+}
```

Comment:
No translated error? Does it occurs only on the server side and didn't shows at the user interface?

Reply:
No invoices will created by customers.

Reply:
Translations were added. Thanks!